### ISSUE TITLE
Files app icon vs. folder icons: different aspect ratio

### ISSUE THREAD

**Issue by matthewpaulthomas** [snwh/suru-icon-theme#96]
yaru-theme-* 18.10.4, Ubuntu Cosmic

The “Files” app icon contains a folder, where the solid part (not including shadows) is 32px ✕ 28px, giving an aspect ratio of about 1.14.

In Files windows, however, at the default zoom level, a folder icon has a solid part of (84px ✕ 78px, giving an aspect ratio of about 1.08.

![icons for folders](https://user-images.githubusercontent.com/19801137/46257774-39a56500-c4b7-11e8-84b8-285f2d4d880c.png)

This is noticeably different. The app icon looks more closer to a real folder that would hold A4 or US Letter paper. Matching that aspect ratio, at a width of 84px, would mean the folders should be 74px high.

[Adapted from [yaru#864](https://github.com/ubuntu/yaru/issues/864).]

**Comment by snwh** [comment]
@matthewpaulthomas this seems a wee bit nit-picky 😉 

I care less about exact accuracy to real-world counterparts of icon metaphors than I do what "looks better" (which is pretty subjective, I know) but the larger "aspect ratio" of the folder icons fills the icon canvas to appear more balanced (it also gives me more space to draw the emblems on the folders).

**Comment by matthewpaulthomas** [comment]
I agree that it isn’t necessary for icons to match the aspect ratio of real-world counterparts. I mentioned it only because one of them already does, demonstrating that it’s practical.

Another way of reconciling them would be to make the Files app icon a pair of diagonally overlapping folders, rather than a single folder.
